spec/components/previews/provider_interface/course_summary_component_preview.rb

```diff
@@ -0,0 +1,7 @@
+module ProviderInterface
+  class CourseSummaryComponentPreview < ViewComponent::Preview
+    def course_details
+      render CourseSummaryComponent.new(course_option: CourseOption.all.sample)
```

Comment:
I did not realise component previews were connecting to a prepopulated database! Is it worth limiting this? e.g. `CourseOption.limit(10).sample` ?